roles/cs.aws-rds/tasks/main.yml

```diff
@@ -16,27 +16,36 @@
     tags: "{{ aws_tags_default }}"
     params: "{{ aws_rds_param_group_params | default(omit) }}"
 
+- name: Create RDS encryption key
+  aws_kms:
+    alias: "{{ aws_rds_encryption_key_name }}"
+    region: "{{ aws_region }}"
+    tags: "{{ aws_tags_default | combine(aws_tags_kms_database) }}"
+  when: aws_rds_storage_encrypt and aws_rds_dedicated_encryption_key
+
 - name: Create MySQL/MariaDB RDS
-  rds:
-    command: create
+  rds_instance:
     region: "{{ aws_region }}"
     publicly_accessible: true
-    instance_name: "{{ aws_rds_instance_name }}"
-    db_engine: "{{ aws_rds_db_engine }}"
-    parameter_group: "{{ aws_rds_param_group_name }}"
+    db_instance_identifier: "{{ aws_rds_instance_name }}"
+    engine: "{{ aws_rds_db_engine }}"
+    db_parameter_group_name: "{{ aws_rds_param_group_name }}"
     engine_version: "{{ aws_rds_db_engine_version }}"
-    size: "{{ aws_rds_storage_size }}"
-    instance_type: "{{ aws_rds_instance_type }}"
-    username: "{{ mageops_mysql_root_user }}"
-    password: "{{ mageops_mysql_root_pass }}"
-    subnet: "{{ aws_rds_subnet_group_name }}"
-    vpc_security_groups: "{{ aws_rds_security_groups }}"
-    backup_retention: "{{ aws_rds_backup_retention }}"
-    backup_window: "{{ aws_rds_backup_window }}"
-    maint_window: "{{ aws_rds_maintenance_window }}"
-    upgrade: "{{ aws_rds_auto_upgrade }}"
+    allocated_storage: "{{ aws_rds_storage_size }}"
+    db_instance_class: "{{ aws_rds_instance_type }}"
+    master_username: "{{ mageops_mysql_root_user }}"
+    master_user_password: "{{ mageops_mysql_root_pass }}"
+    db_subnet_group_name: "{{ aws_rds_subnet_group_name }}"
+    vpc_security_group_ids:
+     - "{{ aws_rds_security_groups }}"
```

Comment:
OK, now I see. I don't like this change. It is misleading. Please revert back to an array.